app.js

```diff
@@ -1,11 +1,60 @@
 /* Imports */
 
 /* Get DOM Elements */
-
+const affirmmeBtn = document.getElementById('affirm me');
+const promptSection = document.getElementById('prompt');
+const affirmationSection = document.getElementById('affirmation');
+const answersP = document.getElementById('answers');
+const resetBtn = document.getElementById('reset');
+const submitBtn = document.getElementById('submit-button');
 /* State */
 
 /* Events */
+submitBtn.addEventListener('click', () => {
+    // hide prompt
+    // display image
+    toggleSections();
+    // pick a random number betwen 0 and length of the answers array - 1
+    const randomInt = Math.floor(Math.random() * answers.length);
+    // console.log(randomInt);
```

Comment:
In the future - you may consider deleting these console.log comments once you've verified functionality.